Remove Go-NEB

luixxiul opened the pull request:
It does not feel nice to remove components in a row, but it does not make sense to keep what's been deprecated by the upstream in the playbook, and reducing tech debt will improve quality of the playbook down the road.

bblacher commented:
A bit off-topic, but does it still make sense to keep the sliding sync proxy around?

luixxiul commented:
If there is not something which uses it then it should be fine to remove it, but I am not sure if there is one or not.

spantaleev commented:
Yes, sliding-sync should be fine to remove. It's long been replaced with native sliding sync.

Commit:
Remove Go-NEB

Reuse https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/096b85bd966931c6ddca8196829c4b99f2fb664f/docs/configuring-playbook-email2matrix.md

Signed-off-by: Suguru Hirahara <did:key:z6MkvVZk1A3KBApWJXv2Ju4H14ErDfRGxh8zxdXSZ4vACDg5>
